cmd/download.go

```diff
@@ -78,33 +78,38 @@ func get(src, dest string, mode getter.ClientMode, cleanGitFolder bool) error {
 	if err != nil {
 		return err
 	}
+
 	client := &getter.Client{
 		Src:  src,
 		Dst:  dest,
 		Pwd:  pwd,
 		Mode: mode,
 	}
-	logrus.Debugf("let's get %s -> %s", src, dest)
 
-	gitFolder := fmt.Sprintf("%s/.git", dest)
-	if cleanGitFolder {
+	logrus.Debugf("let's get %s -> %s", src, dest)
 
-		if _, err := os.Stat(dest); !os.IsNotExist(err) {
-			logrus.Infof("%s already exists! removing it", dest)
-			err = removeDir(dest)
-			if err != nil {
-				logrus.Error(err)
-				return err
-			}
+	if _, err := os.Stat(dest); !os.IsNotExist(err) {
+		logrus.Infof("%s already exists! removing it", dest)
+		err = removeDir(dest)
+		if err != nil {
+			logrus.Error(err)
+			return err
 		}
 	}
 
 	humanReadableDownloadLog(src, dest)
-	_ = client.Get()
+
+	err = client.Get()
+	if err != nil {
+		return err
+	}
+
```

Comment:
so, the bug was that we were failing when the wrong tag/branch/version was specified but we were failing silently? 🙃

Reply:
Yes, basically! :D